extension/src/popup/components/account/AssetDetail/index.tsx

```diff
@@ -80,7 +80,7 @@ export const AssetDetail = ({
     selectedAsset,
   });
 
-  const stellarExpertUrl = getStellarExpertUrl(networkDetails.network);
+  const stellarExpertUrl = getStellarExpertUrl(networkDetails);
```

Comment:
`getStellarExpertUrl` now needs all of `networkDetails` to determine what network a user is on